api/capi/include/nnstreamer-single.h

```diff
@@ -74,6 +74,7 @@ typedef void *ml_single_h;
  * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
  * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
  * @retval #ML_ERROR_STREAMS_PIPE Failed to start the pipeline.
+ * @retval #ML_ERROR_PERMISSION_DENIED The application does not have the privilege to access to the media storage or external storage.
```

Comment:
If the function check_feature_state() returns ML_ERROR_PERMISSION_DENIED, then do we add this error code in all C-API functions?